homeassistant/components/openai_conversation/__init__.py

```diff
@@ -63,15 +71,25 @@ async def async_process(
         self, user_input: conversation.ConversationInput
     ) -> conversation.ConversationResult:
         """Process a sentence."""
-        model = DEFAULT_MODEL
+        raw_prompt = self.entry.options[CONF_PROMPT]
+        raw_contprompt = self.entry.options[CONF_CONTINUED_PROMPT]
```

Comment:
Let's remove continued prompt. It adds complexity that's not needed.

Reply:
Removed